apps/allocations/app/components/Card/Budget.js

```diff
@@ -37,37 +37,20 @@ const Budget = ({
   const edit = () => {
     onEdit(id)
   }
-  // const deactivate = () => {
-  //   onDeactivate(id)
-  // }
 
-  const reactivate = () => {
-    onReactivate(id)
+  const deactivate = () => {
+    onDeactivate(id)
   }
+
   const tokensSpent = BigNumber(amount).minus(remaining)
-  if (inactive) {
-    return (
-      <Wrapper
-        name={name}
-        theme={theme}
-        menu={
-          <ContextMenuItem onClick={reactivate}>
-            <IconView />
-            <ActionLabel>Reactivate</ActionLabel>
-          </ContextMenuItem>
-        }
-      >
-        <StatsValueBig theme={theme}>
-          <Text>Inactive</Text>
-        </StatsValueBig>
-      </Wrapper>
-    )
-  }
 
   return (
     <Wrapper
       name={name}
       theme={theme}
+      amount={amount}
+      symbol={token.symbol}
+      inactive={inactive}
       menu={
```

Comment:
I think we only pass this `menu` prop in once, now. Maybe it makes sense to get rid of the prop and define it inline. Agree?

Reply:
I'm hesitant to do this as we'd have to pass 3 functions as props instead of 1 node. Is that what you want?

Reply:
Ah, nevermind, no need to bother with this right now. Thanks!